h/static/styles/partials-v2/_search.scss

```diff
@@ -279,6 +281,20 @@ $stats-icon-column-width: 20px;
   width: 9px;
 }
 
+.search-bucket-stats-filler {
+  flex-grow: 1;
+}
+
+.search-bucket-stats__collapse-view {
+  @include link;
+  @include link-underline(dotted);
```

Comment:
There is a mock here https://github.com/hypothesis/product-backlog/issues/79#issuecomment-267441967 and it seems that the implemented design does not match it. Note, this comment: https://github.com/hypothesis/product-backlog/issues/79#issuecomment-267371896 has the icon link as well.